content/_sitedata/cta/default.md

```diff
@@ -0,0 +1,14 @@
+---
+templateEngineOverride: njk
+title: Want to chat about your web project or training?
+face: miriam
+summary: |
+  Contact us for custom web app design & development
+  with our team of Python, UX, and front-end experts --
+  or a 1-3 day workshop with Miriam Suzanne.
```

Comment:
Is there any way to re-word this to not have so many horizontal lines in one small area?
![Screen Shot 2020-01-08 at 3 38 02 PM](https://user-images.githubusercontent.com/1581694/72018558-897fe980-322d-11ea-8882-62ac9e7f1903.jpg)

Reply:
@SondraE I made a new trello card for considering this…